tag_release.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Read target version from command line
+TARGET_VERSION=$1
+CURRENT_VERSION=$(sed -n 's/.*\version\": \"\([^"]*\).*$/\1/p' package.json)
```

Comment:
This line is not working for me
![Screenshot from 2024-01-29 12-03-23](https://github.com/iotaledger/explorer/assets/88061365/e470fa8d-612a-4876-8ecd-2702a095d787)

Reply:
```suggestion
CURRENT_VERSION=$(sed -n 's/.*"version": "\([^"]*\)".*/\1/p' package.json)
```